CHANGES.md

```diff
@@ -66,6 +66,7 @@
 ## New Features / Improvements
 
 * 'Manage Clusters' JupyterLab extension added for users to configure usage of Dataproc clusters managed by Interactive Beam (Python) ([BEAM-14130](https://issues.apache.org/jira/browse/BEAM-14130)).
+* Go SDK users may now write self-checkpointing Splittable DoFns to read from streaming sources. **Note: this feature is not yet fully validated and should be treated as experimental in this release.** ([BEAM-11104](https://issues.apache.org/jira/browse/BEAM-11104))
```

Comment:
This generally LGTM, could you add a comment in the linked JIRA that explains what validation has been done/needs to be done as well? If I came across this as a user, I'd probably want to have a larger understanding of what this actually means, but the full explanation doesn't belong in the CHANGES.md

Reply:
I updated the ticket to have that information. Largely just the lack of an integration test for it right now, will be poking around at that this week since Impulses apparently count as DataSources. We should be able to mock together a somewhat generic test case assuming the runners play nice and terminate the pipeline when appropriate.